test/unittest/unittest_blas_kernels_cl.cpp

```diff
@@ -32,13 +32,58 @@ static void setUpGpuContext() {
   ac.initBlasClKernels();
 }
 
-TEST(blas_kernels, dotCL_sgemv) {
+TEST(blas_kernels, dotCL_sgemv_M_1_1) {
```

Comment:
One more Question)
I brought this PR to my local environment for testing. 

It seems that even when setting the ```enable-opencl``` option to true, cannot reach the corresponding unit test.
Is it correct that the current test file is not included in any of the meson.build files within the unittest folder?

Reply:
And when add 
```
if get_option('enable-opencl')
  test_target += [['unittest_blas_kernels_cl', []]]
endif
```
this to unittest/meson.build

i got this build error
```
/home/donghak/Desktop/workspace/nntrainer/nntrainer/opencl/opencl_program.cpp:73:10: error: variable length array ‘binaries_size’ is used [-Werror=vla]
   73 |   size_t binaries_size[num_devices];
      |          ^~~~~~~~~~~~~
/home/donghak/Desktop/workspace/nntrainer/nntrainer/opencl/opencl_program.cpp:98:8: error: variable length array ‘kernel_names’ is used [-Werror=vla]
   98 |   char kernel_names[kernel_names_size];
      |        ^~~~~~~~~~~~
/home/donghak/Desktop/workspace/nntrainer/nntrainer/opencl/opencl_program.cpp:113:18: error: variable length array ‘binaries_ptr’ is used [-Werror=vla]
  113 |   unsigned char *binaries_ptr[num_devices];

```

Reply:
This error should be fixed to enable x86 OpenCL compilation (for tizen build as well). This error may not be detected in Android build. One can face this error, for compilation with PC environment doesn't support dynamic allocation like that. I will make an issue for it :
#2776

Reply:
`unittest_blas_kernels_cl` was written for Android tests. So, it's only present in the Android.mk file.

Reply:
@DonghakPark There was an error in my previous comment. The PR https://github.com/nnstreamer/nntrainer/pull/2727 resolves this issue! But it needs to be rebased. I will update the issue as well. Sorry for confusion

Reply:
I understand. Thank you for your explanations! 👍